app/workers/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -60,6 +61,7 @@ class SubmitForm0781 < Job
       # `file`, which is the generated file location
       def get_docs(submission_id, uuid)
         @submission_id = submission_id
+        @uuid = uuid
```

Comment:
my use case is just adding it as a breadcrumb on the transaction.  Assuming a value is passed, even if not used functionally, it seems like it could be worth logging.  However, this is an intentionally 'dumb' implementation, so I didn't look to hard at how / why that  variable is being passed  ¯\_(ツ)_/¯ I think the comment still seems valid, since my 'use' is really a non-functional, non-use.